cuegui/cuegui/plugins/MonitorCuePlugin.py

```diff
@@ -169,7 +169,19 @@ def __showMenuHandle(self, action):
         """Handles adding or removing shows via the show selection menu
         @type  action: QAction
         @param action: Click action"""
-        if action.isChecked():
+        if action.text() == 'All Shows':
+            try:
+                shows = sorted([show.name() for show in opencue.api.getActiveShows()])
+            except Exception as e:
```

Comment:
Catching `Exception` feels a bit broad here -- if you want to catch API errors maybe catch [`opencue.exception.CueException`](https://github.com/AcademySoftwareFoundation/OpenCue/blob/4af32d5b6c27b49873e17345e51545fcda5b67ff/pycue/opencue/exception.py#L26) instead?

Reply:
Makes sense, will update that now